main.py

```diff
@@ -1,10 +1,49 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as players_file:
+        players = json.load(players_file)
+
+    for player_nickname, player_data in players.items():
+        race_data = player_data["race"]
+        if not Race.objects.filter(name=race_data["name"]).exists():
+            Race.objects.create(
+                name=race_data["name"],
+                description=race_data["description"],
+            )
+        race = Race.objects.get(name=race_data["name"])
```

Comment:
In this case better to use [get_or_create](https://docs.djangoproject.com/en/4.2/ref/models/querysets/#get-or-create) 

The same fix with guild

Reply:
fixed